src/main/java/org/example/animals/DomesticCat.java

```diff
@@ -0,0 +1,18 @@
+package org.example.animals;
+
+public class DomesticCat implements Animals{
```

Comment:
Please, don't forget to format code before committing.
```suggestion
public class DomesticCat implements Animals {
```

Reply:
Sorry. Have done it recently